Fixed unit testing

MarcoLPR opened the pull request:
Added Babel config.
Deleted unused tests.
Fixed tests.

Test to do:
CentralCard
FixedLinearProgress
FormSwitch
MenuList
Navbar
UnoSuspense

![image](https://user-images.githubusercontent.com/25437790/57394154-5f4fc680-718a-11e9-8e18-59529cfa6789.png)

MarcoLPR committed:
Fixed unit testing

Added Babel config.
Deleted unused tests.
Fixed tests.

Test to do:
CentralCard
FixedLinearProgress
FormSwitch
MenuList
Navbar
UnoSuspense

geoperez commented:
Add the pending components in this branch too!

geoperez commented:
I don't like the babel thing, you should add Typescript specs and use probably ts-node. Revert the Babel dependency.

MarcoLPR committed:
WIP

Still have trouble with TextValidator and MenuList tests, I've commented some tests that depends on conditional rendering because the actual version of enzyme isn't rendering correctly these cases.

geoperez commented:
Sync your branch.

geoperez committed:
Merge
